.github/workflows/build_serverless.yml

```diff
@@ -80,7 +79,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x]
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
i'm guessing node 10.x isn't really used anymore?

Reply:
node 10.x is pretty old. I think AWS stopped support for it last year. This wasn't a necessary change but just seemed like a good idea to update for this repo